README.md

```diff
@@ -103,4 +103,10 @@ At this point, the django-shibboleth-remoteuser middleware should be complete.
     )
    ```
 
+### Permission group mapping
+ * It is possible to map a list of attributes to Django permission groups. ```django-shibboleth-remoteuser``` will generate the groups from the semicolon-separated values of this attributes. They will be available in the Django admin interface and you can assign your application permission to them.
```

Comment:
"this attributes" => "these attributes". "application permission" => "application permissions"